README.md

```diff
@@ -151,12 +156,13 @@ enum FlightLogHeader {
         flightController: { serialNumber: 'N/A', firmware: '02.04.20.50' },
         gimbal: { firmware: '01.31.01.67' },
         remoteController: { serialNumber: '12345678', firmware: '1.2.0.17' },
-        camera: { serialNumber: 'N/A' }
+        camera: { serialNumber: 'N/A' },
+        user: { userId: '5fac2b9256338b4d7fd43852', organizationId: '5fac2c0dd27ea76caff42e9f' }
     },
     rows:  [
         { 'Date/Time (GMT)': '06/04/2018 20:32:36',
         'Elapsed Time (sec)': 6.675,
-        Info: 'Mission did take off',
+        Info: "{'message': 'Mission did take off'}",
```

Comment:
I'll get you an example, but this would be `[{'message': 'Mission did take off'}]`

Reply:
@fishstix-dronedeploy Updated.